performance_inference.md

```diff
@@ -0,0 +1,70 @@
+|            | 2048               | 2048                 | 2                 | 78         | 958 tokens/sec   |`QUANT_CONFIG=./quantization_config/maxabs_quant.json   TQDM_DISABLE=1 python3 ../gaudi_spawn.py --use_deepspeed --world_size 2 run_generation.py --model_name_or_path meta-llama/Llama-2-70b-hf --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --limit_hpu_graphs --bucket_size=128 --bucket_internal --max_new_tokens 2048 --max_input_tokens 2048 --bf16 --batch_size 78 --disk_offload --use_flash_attention --flash_attention_recompute` |
+| Mistral 7b Instruct | 128                | 128                  | 1                 | 896        | 13112 tokens/sec |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 896 --max_new_tokens 128 --max_input_tokens 128 --limit_hpu_graphs`                                                                                                                                                |
+|            | 128                | 2048                 | 1                 | 120        | 7947 tokens/sec  |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 120 --max_new_tokens 2048 --max_input_tokens 128 --limit_hpu_graphs  --bucket_internal --bucket_size 128`                                                                                                          |
+|            | 2048               | 128                  | 1                 | 120        | 1360 tokens/sec  |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 120 --max_new_tokens 128 --max_input_tokens 2048 --limit_hpu_graphs`                                                                                                                                               |
+|            | 2048               | 2048                 | 1                 | 44         | 3143 tokens/sec  |`QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_generation.py   --model_name_or_path mistralai/Mistral-7B-Instruct-v0.2 --attn_softmax_bf16 --use_hpu_graphs --trim_logits --use_kv_cache --reuse_cache --bf16 --batch_size 44 --max_new_tokens 2048 --max_input_tokens 2048 --limit_hpu_graphs  --bucket_internal --bucket_size 128`                                                                                                          |
```

Comment:
I get errors for all these commands when launching the measures with `QUANT_CONFIG=./quantization_config/maxabs_measure.json`. I followed the instructions given in the setup section. I'm on Gaudi2 1.16. What am I doing wrong?

Reply:
I found that we need to remove --limit_hpu_graphs when measuring... maybe we should add that fact in the readmes somewhere

Reply:
We will publish these instruction on the intel website instead.   We will follow up on why the --limit_hpu_graphs is causing an issue.